packages/stats/gbstats/bayesian/tests.py

```diff
@@ -223,3 +240,129 @@ def get_risk(mu, sigma) -> List[float]:
         risk_ctrl = float((1.0 - prob_ctrl_is_better) * mn_pos)
         risk_trt = -float(prob_ctrl_is_better * mn_neg)
         return [risk_ctrl, risk_trt]
+
+
+class Bandits(object):
```

Comment:
What does `object` do here?

Reply:
it clutters the screen :)

removed `object`